vizro-core/docs/pages/user_guides/custom_tables.md

```diff
@@ -1,13 +1,13 @@
-# How to create custom tables
+# How to create custom Tables
```

Comment:
Would stay consistent with the rest of the docs e.g. "How to create custom charts"
```suggestion
# How to create custom tables
```